e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -58,6 +58,11 @@ export class UIhelper {
     }
   }
 
+  async clickDivByDataTestId(dataTestId: string) {
+    await this.page.waitForSelector(`div[data-testid="${dataTestId}"]`);
+    await this.page.click(`div[data-testid="${dataTestId}"]`);
```

Comment:
Playwright has a method `page.getByTestId()` that does same thing without providing the locator.